blog/26-01-2023 - All you need to know about Celo to Ease your Web3 Development Journey/Web3 Development with Celo

```diff
@@ -0,0 +1,12 @@
+---
+title: All you need to know about Celo to Ease your Web3 Development Journey
+description: In this tutorial, we will introduce you to the Celo blockchain and explain the key features and benefits of using it for your web3 development projects. We will cover the basics of Celo and provide an overview of its technology and ecosystem, as well as discuss the tools and resources available to help you get started. By the end of this tutorial, you will have a better understanding of Celo and how it can support your web3 development journey. Whether you are just starting out or are an experienced developer, this tutorial will provide you with the knowledge you need to get started with Celo.
+authors: 
+  - name: Abdulateef Sanni
+tags: [celosage, celo, beginner, Web3, Web3 development]
+hide_table_of_contents: true
+slug: /tutorials/All-you-need-to-know-about-Celo-to-Ease-your-Web3-Development-Journey
+---
+## Video Link
+[ All you need to know about Celo to ease your web3 development](https://youtu.be/ZpWLdkSTyyc)
```

Comment:
There is one thing that is missing i.e. a thumbnail, kindly add it to the showcase/beginner directory. This will be good to go. Otherwise great work @Latsan